services/dbaas/client.go

```diff
@@ -9,35 +9,40 @@ import (
 	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils/constant"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/meta"
+	mariadb "github.com/ionos-cloud/sdk-go-dbaas-maria"
 	mongo "github.com/ionos-cloud/sdk-go-dbaas-mongo"
 	psql "github.com/ionos-cloud/sdk-go-dbaas-postgres"
 	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils"
 )
 
-type PsqlClient struct {
-	sdkClient *psql.APIClient
+type MariaDBClient struct {
+	sdkClient *mariadb.APIClient
 }
 
 type MongoClient struct {
 	sdkClient *mongo.APIClient
 }
 
-func NewPsqlClient(username, password, token, url, version, terraformVersion string) *PsqlClient {
-	newConfigDbaas := psql.NewConfiguration(username, password, token, url)
+type PsqlClient struct {
+	sdkClient *psql.APIClient
+}
 
-	if os.Getenv(constant.IonosDebug) != "" {
+func NewMariaDBClient(username, password, token, url, version, terraformVersion string) *MariaDBClient {
+	newConfigDbaas := mariadb.NewConfiguration(username, password, token, url)
+
+	if os.Getenv("IONOS_DEBUG") != "" {
```

Comment:
IonosDebug can be used i think

Reply:
@adeatcu-ionos solved in a25d7c6c779850ffeb70f942e90a214806ee271d